upload_movie.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import os
+import pandas as pd
+import re
+import logging
+from imdb import IMDb
+from multiprocessing import Pool
+from pymongo import MongoClient
+
+
+RELEVANT_KEYS = ['imdbID', 'title', 'genres', 'votes', 'plot', 'synopsis']
+
+
+def get_connection_url(user, password, url):
```

Comment:
This function should be somewhere else as it will be reused many times. I suggest a new file called mongo_utils or something like this